salt/returners/highstate_return.py

```diff
@@ -448,6 +457,12 @@ def _produce_output(report, failed, setup):
         sender = setup.get('smtp_sender', '')
         recipients = setup.get('smtp_recipients', '')
 
+        host = setup.get('smtp_server', '')
+        port = int(setup.get('smtp_port', 0))
```

Comment:
Default to 25 perhaps?

Reply:
That was my first thought, however, since the port value wasn't set before and the default value in the `smtplib` is `0` (see [official documentation](https://docs.python.org/3.6/library/smtplib.html#module-smtplib)), I thought that for backward compatibility it was better to set the same vale as the default one.

I was really tempted for setting the value to `25` too.

Maybe this could be discussed.

Reply:
I don't think we have backward compat issues here since we're introducing a new feature and in other places in salt we default to 25. IMHO, that is what we should do here.